tests/_cli_tools/get_tso.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
we can make `cdc` support this command directly? like `cdc get_tso`  or `cdc ctrl get_tso`